### PR TITLE
In 141914825 modify intake to query tpt and intake apis

### PR DESCRIPTION
**Pull request by brunoOSI**

### CODE REVIEW

File: config/initializers/api.rb [hunk 1]
```diff
@@ -5,16 +5,37 @@
...
+    api_connection = ::API.intake_api_connection
+
+    if ENV.fetch('TPT_API_URL', false) && (url.match %r{/api/v1/people_search})
+      api_connection = ::API.tpt_api_connection
+    end
+
+    api_connection
+  end
+
   def self.make_api_call(url, method, attributes = nil)
```
Comment (ThomasOSI) [inline config/initializers/api.rb]: Could the `attributes` argument be changed to something more meaningful, like `payload`?

File: config/initializers/api.rb [hunk 2]
```diff
@@ -5,16 +5,37 @@
...
+  def self.intake_api_connection
+    @connection ||= Faraday.new(url: ENV.fetch('API_URL')) do |connection|
+      ::API.connection_settings connection
     end
   end
 
+  def self.fetch_api_connection(url)
+    api_connection = ::API.intake_api_connection
+
+    if ENV.fetch('TPT_API_URL', false) && (url.match %r{/api/v1/people_search})
```
Comment (ThomasOSI) [inline config/initializers/api.rb]: Add description of `TPT_API_URL` to the wiki at https://github.com/ca-cwds/intake/wiki/Environment-Variables.

Curious why the fallback is only for people search.

Comment (brunoOSI) [inline config/initializers/api.rb]: At this moment this is the only endpoint that needs to be redirected to TPT. When that changes in the future with more resources we can modify this method

Comment (brunoOSI) [inline config/initializers/api.rb]: Added the description of TPT_API_URL to wiki, thanks.

**Pull request merged**